api/Dockerfile

```diff
@@ -56,9 +56,12 @@ RUN usermod -a -G clamav python
 RUN chown -R python:python ${FUNCTION_DIR}
 
 # Setup ClamAV
-RUN mkdir -p /tmp/clamav \
+RUN mkdir -p /clamav \
+ && mkdir -p /tmp/clamav \
```

Comment:
You can get rid of this line since the one below it will create the missing `clamav` dir thanks to the `-p` flag:
```dockerfile
&& mkdir -p /tmp/clamav/quarantine
```

Reply:
Good catch 🎉